l2geth/core/vm/operations_acl.go

```diff
@@ -24,47 +24,171 @@ import (
 	"github.com/ethereum-optimism/optimism/l2geth/params"
 )
 
```

Comment:
Another difference: we use `big.Int` while upstream uses `uint256.Big`

Reply:
https://github.com/holiman/uint256
Our version of geth is old and doesn't use this